docs/source/start-routingspec.rst

```diff
@@ -662,22 +717,200 @@ so consider the name before saving.
++-------------------------+------------------------------------------------+
+|Destination              |consumer (pipe, database, file, etc)            |
++-------------------------+------------------------------------------------+
+|Output Format            |set output format                               |
++-------------------------+------------------------------------------------+
+|Date/Time - Since/Util   |define time window for message retrieval        |
++-------------------------+------------------------------------------------+
+|Platform Selection       |Specify a platform or network list              |
++-------------------------+------------------------------------------------+
+
+The **Data Source** will populate with a drop down menu consisting of the
+ source or sources added in step 3.  The **Destination** will populate with 
+ some options including but not limited to the database, file and pipe.  
+ To set up a routing spec to go into the database, select the applicable 
+ database.  The **Date/Time** format is only applicable for sources
+ that are not static.  For example, sources of type lrgs or abstract web
+ will require a look back time window to be defined.  The longer the look 
+ back the longer the routing spec will take to run, because there will
+ presumably be more data.  It is good practice to take into consideration
+ how frequently the source is updated, the resolution of the data, 
+ how frequently you are planning to run the routing spec, and how many
+ platforms the routing spec will include.  The **Platform Selection** 
+ is where the user will need to either specify a platform or network list.
+ Recall that a network list is a group of platforms.  
 
 .. image:: ./media/start/routingspec/im-28-rs-source.JPG
    :alt: new list window
    :width: 500
 
+NOTE - there are some unique criteria that need to be selected given 
+certain types of sources.  For example, when the source type is a 
+directory, then the property "oneScanOnly" must be defined as True.
+
+TIP - It is good practice to define the **Destination** as pipe before
+putting a routing spec in operation.  This allows the user to test the 
+routing spec without potentially overwriting or accidentally populating
+another time series in the database.  
+
+Remember to save the changes.  Now you have set up a routing spec.  
+This means that assuming all the steps above have been completed
+or checked off, then a user can now run the routing spec. Routing
+specs can be run manually, irregardless of whether any continuous 
+OpenDCS process is running.  If the destination is a database, the
+database must be up and running.  
+
+
+How do I run or execute the Routing Spec?
+-----------------------------------------
+
+Thus far, we have used the OpenDCS GUI to set up the routing spec.  
+However, running the routing spec 'manually' requires running the
+OpenDCS command 'rs'.  In the following examples, imagine there is 
+a routing spec named 'mySpec'.
+
```

Comment:
```suggestion
Thus far, we have used the OpenDCS GUI to set up the routing spec.  
However, running the routing spec 'manually' requires running the
OpenDCS command 'rs'.  In the table below each option for the rs command is demonstrated with a routing spec named 'mySpec'.   In practice multiple options 
are often used to achieve the desired result such as using both the 'Since Time' and 'Until Time' options.

```